docs/cis_self_assessment.md

```diff
@@ -1349,14 +1230,13 @@ fi
 **Audit Execution:**
 
 ```
```

Comment:
1.2.34 deviates far from the norm. it has the old style audit script and audit execetution. Is this inentoinal?

Also lloks like remediation is in there twice

Reply:
for the audit for this one, I would just give a command to cat the encryption config file and tell them to verify that the provider is aescbc